client/app/bundles/course/level/reducers/level.js

```diff
@@ -0,0 +1,81 @@
+import actionTypes from 'course/level/constants';
+
+const initialState = {
+  levels: [],
+  isLoading: false,
+  isSaving: false,
```

Comment:
This boolean is currently unused. You should use it in the level page to disable add/edit/delete of levels while waiting for the response from the server.

Reply:
Page elements are now disabled after clicking 'Save', while waiting for the response from the server.